README.rst

```diff
@@ -112,7 +112,6 @@ Scenario decorator
 The scenario decorator can accept the following optional keyword arguments:
 
 * ``encoding`` - decode content of feature file in specific encoding. UTF-8 is default.
```

Comment:
let's remove this too? I don't think anybody uses any other encoding

Reply:
Okay.. Leaving the code part intact if you want to change it in your branch.